lightkurve/search.py

```diff
@@ -191,12 +216,27 @@ def download_all(self, quality_bitmask='default', download_dir=None):
         if download_dir is None:
             download_dir = self._default_download_dir()
 
-        path = Observations.download_products(self.table, mrp_only=False,
-                                              download_dir=download_dir)['Local Path']
+        # if table contains TESScut search results, download cutouts
+        if 'TESScut' in self.table[0]['productFilename']:
+            if cutout_size is None:
+                cutout_size = 5
+            elif cutout_size < 0:
+                raise ValueError('`cutout_size` must be positive.')
+            elif cutout_size > 100:
+                warnings.warn('Cutout size is large and may take a few minutes to download.')
+            path = [self._fetch_tesscut_path(t, s, download_dir, cutout_size)
+                    for t,s in zip(self.table['target_name'], self.table['sequence_number'])]
+        else:
+            if cutout_size is not None:
+                warnings.warn('`cutout_size` can only be specified for TESS '
+                              'Full Frame Image cutouts.', LightkurveWarning)
+
```

Comment:
There's a bit of code duplication here between `download` and `download_all`. You could make a helper function to do this, but you don't have to do it for this PR.

Reply:
For example, you could move this `cutout_size` test to the `_fetch_tesscut_path` in order to have it only called once.